test/functional/failing.js

```diff
@@ -83,9 +81,7 @@ describe('Basic Failing Task', function () {
          */
         var workerRunPromise = _Worker.prototype.run.firstCall.returnValue;
         assert.isFulfilled(workerRunPromise);
-        // and, make sure the error module has logged the error
-        assert.ok(_PonosErrorCat.log.calledOnce, 'error.log called');
-        var err = _PonosErrorCat.log.firstCall.args[0];
+        var err = _Worker.prototype._reportError.firstCall.args[0];
```

Comment:
referencing the two lines of code that were removed: previously, it was checked that the log was called only once and then it went to get the error it was called with. With that `calledOnce` check removed, the error that you would get here if it _wasn't_ called is `cannot read property 'args' of undefined`, which is much more cryptic. Let's put back in the `calledOnce` check (with an appropriate message).